### DISCUSSION TITLE
Github sponsor, Kofi, etc

### DISCUSSION THREAD

**Discussion by chrismwiggs** [Geolykt/EnchantmentsPlus#49] [Ideas]
You should set up some way for us to send you a little something for all the work you'd done to support Enchantments plus! Also is there a reason (the original author maybe?) that your fork isn't on spigot? It's a shame how hard it is to find it

**Comment by Geolykt** [comment]
https://www.spigotmc.org/resources/enchantmentplus.84139/ it is on spigot for a long time now, however according to our bstats statistics it isn't used that often (only 13.1% compared to bukkit's 78.1% market share).

About donations, It will probably not happen as while I can work on my things now I certainly cannot in the future. Additionally I lack the underlying payment infrastructure to even set it up in the first place (one day I'll set it up, but it won't be today). The best (and only) way of supporting continued development is to report bugs and maybe to profile the plugin if you really want to.
